code/__DEFINES/clothing.dm

```diff
@@ -94,4 +94,7 @@
 #define MASKCOVERSEYES		2		// get rid of some of the other retardation in these flags
 #define HEADCOVERSEYES		4		// feel free to realloc these numbers for other purposes
 #define MASKCOVERSMOUTH		8		// on other items, these are just for mask/head
-#define HEADCOVERSMOUTH		16
\ No newline at end of file
+#define HEADCOVERSMOUTH		16
+

```

Comment:
I moved the two tint defines here instead of being defined twice in human/life and species.
I renamed TINT_IMPAIRED to TINT_DARKENED to avoid confusion, since this define gives an fullscreen overlay called "DarkMask" and there is a different fullscreen overlay called "vimpaired" (used for nearsighted disabilty).